tests/clipboard/paste.ts

```diff
@@ -65,6 +77,18 @@ test('does not paste when disabled', async () => {
   )
 })
 
+test('does not paste when preventDefault is called', async () => {
+  const {element, getEventSnapshot, user} = setup(`<input />`)
+  element.addEventListener('paste', e => e.preventDefault())
+
+  await user.paste('hi')
+  expect(getEventSnapshot()).toMatchInlineSnapshot(`
+    Events fired on: input[value=""]
+
+    input[value=""] - paste
+  `)
```

Comment:
```suggestion
  const {element, eventWasFired, user} = setup(`<input />`)
  element.addEventListener('paste', e => e.preventDefault())

  await user.paste('hi')
  expect(eventWasFired('paste')).toBe(true)
  expect(eventWasFired('input')).toBe(false)
```
Let's prefer explicit assertions over snapshots.